api/admin/passport.js

```diff
@@ -49,14 +49,19 @@ if (Features.enabled(Features.Flags.FEATURE_AUTH_UAA)) {
     ...config.passport.uaa.options,
     callbackURL: `${config.app.hostname}/admin/auth/uaa/callback`,
     logoutCallbackURL: `${config.app.hostname}/admin/auth/uaa/logout`,
+    passReqToCallback: true,
   };
 
-  const verify = async (accessToken, refreshToken, profile, callback) => {
+  const verify = async (req, accessToken, refreshToken, profile, callback) => {
     try {
-      const user = await verifyUAAUser(accessToken, refreshToken, profile, ['pages.admin']);
-
+      let user = await verifyUAAUser(accessToken, refreshToken, profile, ['pages.support']);
+      if (!user) {
```

Comment:
This block is a bit confusing when using the `verifyUAAUser` function and it will always set the `req.session.role` to some value even if they are not an admin or support user. Maybe we should create a new function in [`uaaStrategy.js`](https://github.com/cloud-gov/pages-core/blob/staging/api/services/uaaStrategy.js) that can simplify this logic. We could create a function to verify and return an object with the user and their role.